### DISCUSSION TITLE
Storybook 7 .mdx Meta docs.container parameter has no affect

### DISCUSSION THREAD

**Discussion by davidlyonsbp** [storybookjs/storybook#24662] [Help]
In Storybook 6, I was wrapping .stories.mdx docs with a custom page template:

```jsx
<Meta
  title="Components/Buttons/Overview"
  parameters={{
    docs: {
      container: ({ children, ...rest }) => (
        <DocsContainer {...rest}>
          <CustomMdxDocsTemplate heading="Buttons">
            {children}
          </CustomMdxDocsTemplate>
        </DocsContainer>
      ),
    },
  }}
/>
```
In Storybook 7, when renaming .stories.mdx to .mdx, the Meta docs.container parameter no longer has an affect.

Setting a global docs.container parameter in preview.tsx has an affect, but can't pass in props unique to each mdx page, such as heading="Buttons"

Is there another way to do this with either the .mdx or the global docs.container ?

**Comment by benkates** [comment]
Also would like this added!
